app/(drawer)/(tabs)/library/music/playlists/[playlistId]/index.tsx

```diff
@@ -0,0 +1,172 @@
+  const { loadTrackList, currentTrackListId } = useMusicPlayer();
+  const { state: playbackState } = usePlaybackState();
+  const [moreIsOpen, setMoreIsOpen] = useState(false);
+  const isThisTrackListLoaded = playlistId === currentTrackListId;
+  const isThisTrackListPlaying =
+    isThisTrackListLoaded && playbackState !== State.Paused;
+
+  const { data: playlistTracks = [] } = useQuery({
+    queryKey: [playlistId],
+    queryFn: () => getPlaylist(playlistId as string),
+  });
+  const { height } = useMiniMusicPlayer();
+
+  const handleRowPress = async (index: number) => {
+    await loadTrackList({
+      trackList: playlistTracks,
+      trackListId: playlistId as string,
+      startIndex: index,
+      playerTitle: playlistTitle as string,
+    });
+  };
+
+  const handlePlayPausePress = () => {
+    if (isThisTrackListLoaded) {
+      return togglePlayPause();
+    }
+
+    return loadTrackList({
+      trackList: playlistTracks,
+      trackListId: playlistId as string,
+      startIndex: 0,
+      playerTitle: playlistTitle as string,
+    });
+  };
+
+  const handleMorePress = () => {
+    setMoreIsOpen(true);
+  };
+
+  return playlistTracks ? (
+    <View style={{ height: "100%", paddingTop: 8, gap: 8 }}>
+      <View
+        style={{
+          flexDirection: "row",
+          alignItems: "center",
+          gap: 24,
+          padding: 8,
+        }}
+      >
+        {!!playlistTracks && (
+          <PlayPauseTrackButton
+            size={40}
+            color={brandColors.pink.DEFAULT}
+            type={isThisTrackListPlaying ? "pause" : "play"}
+            onPress={handlePlayPausePress}
+          />
+        )}
+        <Text
+          style={{
+            fontSize: 24,
+            textAlign: "left",
+            flex: 1,
+          }}
+          numberOfLines={3}
+          bold
+        >
+          {playlistTitle}
+        </Text>
+        <Pressable onPress={handleMorePress}>
+          <MaterialCommunityIcons
+            name="dots-horizontal"
+            size={24}
+            color={colors.text}
+          />
+        </Pressable>
+      </View>
+      <FlatList
+        data={playlistTracks}
+        contentContainerStyle={{ flexGrow: 1 }}
+        renderItem={({ item, index }) => {
+          const { id, title, artist } = item;
+          const isLastRow = index === playlistTracks.length - 1;
+          const marginBottom = isLastRow ? height + 16 : 16;
+
+          return (
+            <TouchableOpacity onPress={() => handleRowPress(index)}>
+              <View
+                style={{
+                  flexDirection: "row",
+                  marginBottom,
+                  alignItems: "center",
+                  gap: 4,
+                }}
+              >
+                {/* TODO - swap placeholder with artwork of first track */}
+                <LogoIcon fill="white" width={60} height={60} />
+                {/* <SquareArtwork size={60} url={artworkUrl} /> */}
```

Comment:
Should be able to get the track artwork from the item here:
```suggestion
                <SquareArtwork size={60} url={item.artworkUrl} />
```